functions/go/apply-replacements/main.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/GoogleContainerTools/kpt-functions-sdk/go/fn"
+	"sigs.k8s.io/kustomize/api/filters/replacement"
+	"sigs.k8s.io/kustomize/api/types"
+	"sigs.k8s.io/kustomize/kyaml/yaml"
+)
+
+func main() {
+	if err := fn.AsMain(fn.ResourceListProcessorFunc(ApplyReplacements)); err != nil {
+		os.Exit(1)
+	}
+}
+
+func ApplyReplacements(rl *fn.ResourceList) error {
```

Comment:
Things under the `main` package can not be exported by other packages. We should move the business logic to a separate package.

`main` package can be as simple as
```
func main() {
	if err := fn.AsMain(fn.ResourceListProcessorFunc(anotherpkg.ApplyReplacements)); err != nil {
		os.Exit(1)
	}
}
```

Reply:
> Things under the main package can not be exported by other packages. We should move the business logic to a separate package.

TIL. Thank you, done! Moved business logic to `replacements` package.

Reply:
Fixed, thanks for the catch!